queue/python/queue.py

```diff
@@ -0,0 +1,33 @@
+class Queue:
+    def __init__(self):
+        self.queue=[]
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpkyj0v92m/queue/python/queue.py
+++ b/tmp/tmpkyj0v92m/queue/python/queue.py
@@ -1,6 +1,6 @@
 class Queue:
     def __init__(self):
-        self.queue=[]
+        self.queue = []
 
     def is_empty(self):
         return self.queue==[]
```